server/tools/jgroups.sh

```diff
@@ -27,4 +27,19 @@ if [ -n "$JGROUPS_DISCOVERY_PROTOCOL" ]; then
     else
        $JBOSS_HOME/bin/jboss-cli.sh --file="/opt/jboss/tools/cli/jgroups/discovery/default.cli" >& /dev/null
     fi
+
+   # How many owners / replicas should our distributed caches have. If <2 any node that is removed from the cluster will cause a data-loss!
+   # As it is only sensible to replicate AuthenticationSessions for certain cases, their replication factor can be configured independently
+   if [ -n "$CACHE_OWNERS_COUNT" ]; then
```

Comment:
I believe you left this modification by a mistake? All the changes seems to be implemented in `infinispan.sh`. If I'm right, could you please update this PR once again and remove this part?

Reply:
@slaskawi  .. argh sorry about that. Fixed it.